src/common/meta/Cargo.toml

```
@@ -6,6 +6,7 @@ license.workspace = true
 
 [features]
 testing = []
+pg_kvbackend = []
```

```suggestion
pg_kvbackend = ["deps:tokio-postgres"]
```
optional deps: https://doc.rust-lang.org/cargo/reference/features.html#optional-dependencies